### DISCUSSION TITLE
anomalib cannot be properly verified during training

### DISCUSSION THREAD

**Discussion by chenmo2** [open-edge-platform/anomalib#1364] [Q&A]
Brother training encountered the problem of verification can not be carried out normally, has been unable to find the reason, what is the solution
![image](https://github.com/openvinotoolkit/anomalib/assets/144789079/7d8875ca-0959-4c7f-9267-bc102bd3a83b)

There are also issues with setting up the go to openvino model
![image](https://github.com/openvinotoolkit/anomalib/assets/144789079/d601e3ac-1fb5-4228-9396-b4b6a5fe5977)
![image](https://github.com/openvinotoolkit/anomalib/assets/144789079/26dfb78e-00b5-440f-9509-86567d3d7721)

**Comment by samet-akcay** [comment]
@chenmo2, can you please upload your config file here as well? With this screenshot, it is really hard to understand what/how you want to achieve

**Comment by samet-akcay** [comment]
Something like this please
https://github.com/openvinotoolkit/anomalib/issues/new?assignees=&labels=&projects=&template=bug_report.yaml&title=%5BBug%5D%3A+
